### PR TITLE
Add prettier + linting fixes

### PR DESCRIPTION
**Pull request by sabeurthabti**
A lot of file changes.

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32823

**Comment by sabeurthabti** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32825

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32826

**Comment by sabeurthabti** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32827

File: src/components/Audioplayer/RepeatDropdown/index.js [hunk 1]
```diff
@@ -69,44 +70,47 @@ class RepeatButton extends Component {
...
-                      <option key={index} value={index + 1}>
-                        {index + 1}
-                      </option>
-                    );
-                  }
-                  return options;
-                }, [])
-              }
+              {array.reduce((options, ayah, index) => {
+                if (index + 1 < chapter.ayat) {
```
Comment (naveed-ahmad) [inline src/components/Audioplayer/RepeatDropdown/index.js]: `chapter.versesCount`

File: src/components/Audioplayer/RepeatDropdown/index.js [hunk 2]
```diff
@@ -69,44 +70,47 @@ class RepeatButton extends Component {
...
-                      </option>
-                    );
-                  }
-                  return options;
-                }, [])
-              }
+              {array.reduce((options, ayah, index) => {
+                if (
+                  (repeat.from ? repeat.from : 1) < index + 1 &&
+                  index + 1 <= chapter.ayat
```
Comment (naveed-ahmad) [inline src/components/Audioplayer/RepeatDropdown/index.js]: chapter.versesCount

**Comment by sabeurthabti** [comment]
I had a thought, this isn't right. I am changing too many files. I think a better approach we keep this just at the pre-commit. When anyone stages files then prettier will kick in. 

I will close this PR and raise another. 

@mmahalwy @naveed-ahmad

**Comment by naveed-ahmad** [comment]
@sabeurthabti prettify as commit hook makes much more sense.

**Comment by mmahalwy** [comment]
+1 on that! Going to delete this branch

**Pull request closed without merging**